### PR TITLE
Add HTML-CSS files of Portfolio

### PR DESCRIPTION
**Pull request by dumbledorep**
This is a pull request for the Portfolio project containing HTML and CSS files

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  background-color: white;
+  display: flex;
+  height: 620px;
+  flex-direction: column;
+  padding: 20px 20px;
+  border-bottom-left-radius: 25% 50px;
+  margin-bottom: 100px;
+}
```
Comment (mwanawabangona) [inline style.css]: - [ ] `required`
- [ ] Great job so far with everything :clap:  however you have left out the backgroud image. please see screenshots below and kindly add the image

Figma template
Screenshot from 2022-07-05 06-48-01


your template
Screenshot from 2022-07-05 06-47-09

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+
+## Materials required for validation
+
+- follow this link :
+ [HTML and CSS Linters link]( https://github.com/microverseinc/linters-config/tree/master/html-css)
+
+## Screenshot
+
+![readme](../Phil-s-Portfolio/images/phil.png)
+
```
Comment (mwanawabangona) [inline README.md]: - [ ] `required`
- [ ] Kindly fix this broken screenshot for the readme to look even better :v:

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
## Status: Changes Required

 ## Hello there @dumbledorep 👋🏽

###  You have done an amazing job so far! 👏🏽 👏🏽


There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand:


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional briefcase
* ✅  Solid pull request scroll
* ✅  Good use of flexbox 🧐
* ✅  Good quality commits

## Required Changes
 _**Check the comments under the review.**_

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @mwanawabangona in your question so I can receive the notification.
>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
>
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          <div class="mobile-nav"><a href="#portfolio" class="closeMenu">Portfolio</a></div>
+          <div class="mobile-nav"><a href="#about" class="closeMenu">About</a></div>
+          <div class="mobile-nav"><a href="#contact" class="closeMenu">Contact</a></div>
+        </div>
+        <div class="item menu"><a href="#" class="openMenu"><i class="fa fa-bars" aria-hidden="true"></i></a></div>
+      </nav>
+    </section>
+  </header>
+  <main class="main-parts">
+    <section class="headline-section">
```
Comment (danmainah) [inline index.html:44]: @dumbledorep great work implementing the changes requested.

**Review by danmainah — APPROVED** [review]
Hi @dumbledorep , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (dumbledorep) [reply index.html:44]: @danmainah Thank you

**Pull request merged**